unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TransactionHUD/Tests/TransactionHUDTests.cs

```diff
@@ -0,0 +1,127 @@
+using NUnit.Framework;
+using System.Collections;
+using DCL.Controllers;
+using DCL.Helpers;
+using UnityEngine;
+using UnityEngine.TestTools;
+using DCL.TransactionHUDModel;
+
+namespace Tests
+{
+    public class TransactionHUDTests : IntegrationTestSuite_Legacy
```

Comment:
We are moving away form IntegrationTestSuiteLegacy but I don't know if there is a problem if you integrate it!